pages/editions/index.js

```diff
@@ -12,7 +12,7 @@ const PastEditions = () => {
                 <h1 className='text-5xl sm:text-4xl px-10 py-4 rounded-full bg-violet-700 text-white my-4 text-center w-fit mx-auto font-bold'>AsyncAPI Conf</h1>
             </div>
 
-            <div className='w-3/4 sm:w-10/12 my-10 mx-auto grid grid-cols-2 sm:grid-cols-1 gap-[60px]'>
+            <div className='w-3/4 sm:w-10/12 my-10 mx-auto grid grid-cols-2 md:grid-cols-1 lg:grid-cols-1 gap-[60px]'>
```

Comment:
In my opinion just defining it for `lg` will work and then putting `grid-cols-1` for all other displays is better ig.